lib/features/authentification/presentation/vues/ComplementGardien.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:flutter/material.dart';
+
+class ComplementGardien extends StatefulWidget {
+  @override
+  _InterfaceState createState() => _InterfaceState();
+}
+
+class _InterfaceState extends State<ComplementGardien> {
+
+    @override
+    Widget build(BuildContext context) {
+      int largeurEcran = MediaQuery.of(context).size.width.floor();
+      int longueurEcran = MediaQuery.of(context).size.height.floor();
+
+      return Scaffold(
+        appBar: AppBar(
+          backgroundColor: Color(0xFFE5E5E5), // Fond gris pour l'AppBar
+          title: Text(
+            'Compléter mes informations',
+            textAlign: TextAlign.center,
+            style: TextStyle(
+              fontSize: 25,
```

Comment:
Faut rendre les dimensions dynamique pour eviter des overflow sur certaines tailles d'ecran
```suggestion
        fontSize: tailleEcran/x,
```